src/sp/it/pl/gui/infonode/InfoTable.kt

```diff
@@ -0,0 +1,82 @@
+package sp.it.pl.gui.infonode
+
+import javafx.scene.control.Labeled
+import javafx.scene.control.TableView
+import org.reactfx.Subscription
+import sp.it.pl.util.reactive.onChange
+import sp.it.pl.util.text.plural
+
+/**
+ * Provides information about table items and table item selection.
+ *
+ * @param <E> type of table element
+ */
+class InfoTable<E>: InfoNode<TableView<E>> {
```

Comment:
TableInfo sounds more intuitive to me

Reply:
Ah yeah I was questioning the names of the classes myself. The idea was to Have common prefix instead of suffix to help with autocompletion and alphabetical sorting (a good cause). I remain torn on what is the correct solution to the AaaBbb vs BbbAaa problem.